content/Kotlin Scope Functions - Amaury Ricardo.md

```diff
@@ -0,0 +1,107 @@
+## Scope functions in Kotlin
+![Kotlin](images/kotlin.png)
+
+### Objectives.
+. Work with the same context.
```

Comment:
I don't know if this is something in the Android/Kotlin programming but it is not clear to me.